spring-boot-modules/spring-boot-keycloak/spring-boot-mvc-client/src/main/java/com/baeldung/boot/keycloak/client/UiController.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.boot.keycloak.client;
+
+import java.util.Objects;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
+import org.springframework.security.oauth2.core.oidc.user.OidcUser;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class UiController {
+  @GetMapping("/")
+  public String getIndex(Model model, Authentication auth) {
+    model
```

Comment:
Indentation should be 4 spaces

Reply:
The Baeldung code formatter is broken with the current Eclipse version, reason for my IDE to fall back to my default one (Google). Can you point me to a valid formatted?